invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/files_integrity_report/email/files_integrity_report.html

```diff
@@ -3,30 +3,29 @@
 #
 # Copyright (C) 2022 CERN.
 # Copyright (C) 2024 KTH Royal Institute of Technology.
+# Copyright (C) 2025 Northwestern University.
 #
 # Invenio App RDM is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 -#}
 
-{% set BASE_URL = config.SITE_UI_URL %}
-
 {{ _("The following files were flagged as 'unhealthy'. This means that the checksum check failed or timed out. Please take any action if needed.") }}
 
 {% for entry in entries -%}
-{{ _("ID: %s") | format(entry.file.id) }}
-{{ _("URI: %s") | format(entry.file.uri) }}
+{{ _("ID: {}").format(entry.file.id) }}
```

Comment:
@inveniosoftware/translation-maintainers I had to do that for the new tests below (that actually tests the real email)  https://github.com/inveniosoftware/invenio-app-rdm/pull/2988/files#diff-14e94d3905a557d050c7fe81fe86e57efb4693e4a1550d19ed56ab50e04da178R176 to pass. It seems like an ok approach from https://inveniordm.docs.cern.ch/develop/howtos/i18n/ (not a lazy string anymore but that's fine I think). If anything egregiously incorrect, let me know.